Pokemon/Pokemon/PokemonLibrary.m

```diff
@@ -0,0 +1,63 @@
+//
+//  PokemonLibrary.m
+//  Pokemon
+//
+//  Created by Brian Blanco on 7/31/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "PokemonLibrary.h"
+
+NSString *const kName = @"name";
+NSString *const kDescription = @"description";
+NSString *const kIcon = @"icon";
+NSString *const kLargeIcon = @"largeIcon";
+NSString *const kType = @"type";
+
+@implementation PokemonLibrary
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        //used to set up my PoKeMon dictionary
+        _library = @[@{kName: @"Pikachu",
```

Comment:
Also—this class should probably just be creating `PokemonList` objects out of these dictionaries, rather than having to do that externally.